Tools/autotest/common.py

```diff
@@ -719,6 +746,23 @@ def run_cmd(self,
                     raise ValueError()
                 break
 
+    def get_default_mode_string(self):
+        """Return the vehicle default mode string."""
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_GROUND_ROVER:
+            return "MANUAL"
+        if self.mav.mav_type in [mavutil.mavlink.MAV_TYPE_QUADROTOR,
+                                 mavutil.mavlink.MAV_TYPE_HELICOPTER,
+                                 mavutil.mavlink.MAV_TYPE_HEXAROTOR,
+                                 mavutil.mavlink.MAV_TYPE_OCTOROTOR,
+                                 mavutil.mavlink.MAV_TYPE_COAXIAL,
+                                 mavutil.mavlink.MAV_TYPE_TRICOPTER]:
+            return "STABILIZE"
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_FIXED_WING:
+            return "MANUAL"
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_SUBMARINE:
+            return "STABILIZE"
```

Comment:
I believe this should be MANUAL too,
With stabilize the sub autotest times out during one of the first wait_heading() calls